src/extras/module-types.js

```diff
@@ -18,8 +18,9 @@
   systemJSPrototype.fetch = function (url, options) {
     return fetch(url, options)
     .then(function (res) {
+      if (!res.ok)
+        return res;
```

Comment:
This might result in an error later in the code, right? If so, maybe better to throw an error?

Reply:
The SystemJS error handler will still apply and throw the error at https://github.com/systemjs/systemjs/blob/master/src/features/fetch-load.js#L25.

Reply:
Ah I see, thanks